runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -2002,7 +2002,7 @@ mod development {
 
 				env.parachain_state_mut(|| {
 					let pool_id = DEFAULT_POOL_ID;
-					let amount = 10 * dollar(12);
+					let amount = 10 * decimals(12);
```

Comment:
@William maybe you can cherry-pick the commit that introduces these changes to avoid a lot of conflicts in your liquidity-pools tests refactor